raincloudy/faucet.py

```diff
@@ -238,14 +233,15 @@ def watering_time(self):
 
         return watering_time
 
-    @watering_time.setter
-    def watering_time(self, value):
-        """Manually turn on water for X minutes."""
-        return self._set_watering_time(self.id, value)
-
     @property
-    def droplet(self):
-        return None
+    def manual_watering(self):
+        """Return zone manual_mode_on"""
+        return self.lookup_attr('manual_mode_on')
+
+    @manual_watering.setter
+    def manual_watering(self, value):
```

Comment:
I think I may want to call this `manual_watering_time` instead. As it stands now we have a setter for `auto_watering` that takes a `boolean` and `manual_watering` that takes a `string`. That's probably going to be too confusing :/

What do you think @tchellomello

Reply:
@vanstinator I agree.. Let's rename it and then we can a deprecation alias for a few versions or we can add a not as breaking change for our users. I'm down to change it!!

Reply:
Ok so coming back to this I think I actually prefer `manual_watering`. I've got a plan for integrating the new functions into HA and it'll end up really clean.